qemu/patches/5.0.x/0003-virtiofsd-add-rlimit-nofile.patch

```diff
@@ -0,0 +1,148 @@
+From: Stefan Hajnoczi <address@hidden>
```

Comment:
We already have precedent, but since this *is* a security-related PR...

I'm not totally comfortable with this email obfuscation as it's no longer obvious who created the patch. Is there a canonical patchwork URL you could specify in the commit as an alternative?

/cc @stefanha, @dagrh.

Reply:
It's also inconsistent as we don't always hide the addresses - see https://github.com/kata-containers/packaging/pull/1177/files :smile:

Reply:
You can link to and fetch the original patches from QEMU's GitLab:
https://gitlab.com/qemu-project/qemu/-/commit/6dbb716877728ce4eb51619885ef6ef4ada9565f.patch